include/marlin/Configuration_A3ides_2209_MINI_adv.h

```diff
@@ -1348,7 +1348,7 @@
     #define FILAMENT_CHANGE_ALERT_BEEPS 10 // Number of alert beeps to play when a response is needed.
     #define PAUSE_PARK_NO_STEPPER_TIMEOUT // Enable for XYZ steppers to stay powered on during filament change.
 
-    #define PARK_HEAD_ON_PAUSE // Park the nozzle during pause and filament change.
+//    #define PARK_HEAD_ON_PAUSE // Park the nozzle during pause and filament change.
```

Comment:
Screen_printing will be refactored soon (4.1.0), I will keep commented code for reference.
SD card and pause and printing state machine is completely reworked in this pull, maybe the title is confusing. Radek Vana will continue to use new pause in filament load/unlad, PAUSE_FEATURE and NOZZLE_PARK_FEATURE will be completely removed just like HOST_ACTIONS. This is not done in this pull, because we need keep filament load/unload functionality in 4.1.0 beta until new implementation will be ready.

Reply:
Ok!